spec/javascript/components/src/views/ForgotPassword.spec.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { fireEvent, render, screen, wait } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import { Router } from "react-router-dom";
+import { createMemoryHistory } from "history";
+import ForgotPassword from "components/src/views/ForgotPassword/index";
+import UserContext from "components/src/contexts/UserContext"
+
+const mockHistory = createMemoryHistory();
+
+const apiCall = jest.fn().mockReturnValue(Promise.resolve({ data: {} }))
+
+const mockInputEmailEvent = { target: { value: "new@email.com" } };
+
+describe("ForgotPassword component", () => {
+  let view = null;
+
+  beforeEach(() => {
+    view = render(
+      <Router history={mockHistory}>
+        <UserContext.Provider value={{ apiCall: apiCall }}>
+          <ForgotPassword />
+        </UserContext.Provider>
+      </Router>
+    );
+  });
+
+  it("should render without errors", async () => {
+    expect(
+      screen.getByText("REQUEST EMAIL PASSWORD RESET")
+    ).toBeInTheDocument();
+  });
+
+  it("should make an API call when form is submitted", async () => {
+    fireEvent.change(screen.getByPlaceholderText(/Please enter your email address/), mockInputEmailEvent);
+    const button = view.container.querySelector("button");
```

Comment:
non-blocking: I think it's an anti-pattern to use `querySelector` in React testing instead of some of the other React Testing Library methods, but this can be changed later since it's from a previously written test.

Reply:
Thanks, I didn't know about that. Yeah, mainly renaming test files, and making the minimal changes to get them running again.